projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -1439,6 +1439,51 @@ protected void SetPlurality(int? value)
                 Subject.MultipleBirth.Extension.Add(plurality);
             }
         }
+    
+        /// <summary>Patient Fetal Death</summary>
+        /// <value></value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.PatientFetalDeath = true;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Patient Fetal Death: {ExampleBirthRecord.PatientFetalDeath}");</para>
+        /// </example>
+        [Property("PatientFetalDeath", Property.Types.Bool, "PatientFetalDeath", "PatientFetalDeath", false, VR.IGURL.PatientFetalDeath, true, 288)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)", "deceased")]
+
+        public bool? PatientFetalDeath
+        {
+            get
+            {
+                if (Subject != null && Subject.Deceased != null)
+                {
+                    Extension patientFetalDeath = Subject.Deceased.Extension.FirstOrDefault(ext => ext.Url == VRExtensionURLs.PatientFetalDeath);
+                    if (patientFetalDeath != null)
+                    {
+                        bool? fetalDeath = ((FhirBoolean)patientFetalDeath.Value).Value;
+                        return fetalDeath;
+                    }
+                }
+                return null;
+            }
+            set
+            {
+                if (Subject.Deceased == null)
+                {
+                    Subject.Deceased = new FhirBoolean(true);
+                }
+                Subject.Deceased.Extension.RemoveAll(ext => ext.Url == VRExtensionURLs.PatientFetalDeath);
+                if (value == null)
+                {
+                    return;
+                }
+                else
+                {
+                    Extension fetalDeath = new Extension(VRExtensionURLs.PatientFetalDeath, new FhirBoolean(value));
```

Comment:
If the comment above is accurate then rather than `new FhirBoolean(value)`, should it just be `new FhirBoolean(true)`?

Reply:
For patientChild, it should be false (or null). I've moved this property out of NatalityRecord and into FetalDeathRecord and BirthRecord since they will have different default booleans.